api/javatests/org/openyolo/api/CredentialClientTest.java

```diff
@@ -295,31 +293,34 @@ public void getSaveIntent_noKnownProviders() throws Exception {
     }
 
     @Test
-    public void getCredentialFromActivityResult_noExtras() throws Exception {
+    public void testGetCredentialRetrieveResult() throws Exception {
```

Comment:
Please remove test prefix.  It's not standardized in this codebase, but I'd really like to follow a naming pattern for unit tests moving forward. <method_name>_<input_qualifiers>_<expected_result> is the template I use.  What do you think about adopting it?